scripts/ghcr_newest_by_created.py

```
@@ -0,0 +1,144 @@
+                )
+                digest = amd["digest"]
+            else:
+                digest = hdr.get("docker-content-digest") or man.get("config", {}).get(
+                    "digest", ""
+                )
+            cfg_digest = man["config"]["digest"]
+            req = urllib.request.Request(
+                f"https://{GHCR}/v2/{repo}/blobs/{cfg_digest}",
+                headers={"Authorization": f"Bearer {tok}"},
+            )
+            with urllib.request.urlopen(req, timeout=60) as r:
```

Ruff (0.16.1)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)